app/plot_app/configured_plots.py

```diff
@@ -195,12 +195,11 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
     data_plot.change_dataset('vehicle_global_position')
     data_plot.add_graph(['alt'], colors8[2:3], ['Fused Altitude Estimation'])
     data_plot.change_dataset('position_setpoint_triplet')
-    data_plot.add_circle(['current.alt'], [plot_config['mission_setpoint_color']],
-                         ['Altitude Setpoint'])
-    data_plot.change_dataset(actuator_controls_0.thrust_sp_topic)
-    if actuator_controls_0.thrust_z_neg is not None:
-        data_plot.add_graph([lambda data: ('thrust', actuator_controls_0.thrust_z_neg*100)],
-                            colors8[6:7], ['Thrust [0, 100]'])
+    if 'position_setpoint_triplet' in data and 'current' in data['position_setpoint_triplet']:
```

Comment:
Why is this changed?

Reply:
Without this, the Y axis range always starts at 0.

Reply:
Well this isn't correct though. How did you test this?
You probably need to take into consideration the `valid` flag.

Reply:
Do you have a log where this doesn't work?

Reply:
Check any log that has some mission on https://logs.px4.io/browse, eg https://logs.px4.io/plot_app?log=22a7603d-edf4-476a-8b0e-0aa0a73b107b